composer.json

```diff
@@ -22,7 +22,7 @@
         "sonata-project/notification-bundle": "~2.2",
         "sonata-project/easy-extends-bundle": "~2.1",
         "sonata-project/doctrine-extensions": "~1.0",
-        "knplabs/gaufrette": ">=0.1.4",
+        "knplabs/gaufrette": ">=0.1.6",
```

Comment:
no idea as to why - it came from cherry picking the commit https://github.com/Bladrak/SonataMediaBundle/commit/ff199ffb6a80fe293d0bf792cd4acb99ca94301e , which is part of PR https://github.com/sonata-project/SonataMediaBundle/pull/640